requirements/requirements.txt

```
@@ -1,5 +1,5 @@
 asyncio
-deepspeed>=0.12.4
+deepspeed>=0.12.7
```

```suggestion
deepspeed>=0.12.4
```

Reverting this, we will update requirements in another PR before next release of DS + MII.